client/public/index.html

```diff
@@ -1,6 +1,8 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyB37PTAW0K2JAbr9LUANA1ntB0KbV21d50">
```

Comment:
Is it necessary to load the Google Maps script in the `head`? Can we do it with the `async defer` options at the end of our `body`?